### PR TITLE
Add periodic and custom trigger for serverless mesh test

### PR DESCRIPTION
**Pull request by nak3**
This patch adds periodic job for serveless mesh test and custom trigger `/test 4.7-operator-e2e-aws-ocp-47-mesh` to trigger mesh test on serverless-operator repo.

### CODE REVIEW

**Comment by nak3** [comment]
/cc @markusthoemmes @mgencur 

Could you please take a look?

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7-aws-mesh.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      - env: OPENSHIFT_KNATIVE_OPERATOR
+        name: openshift-knative-operator
+      from: serverless-operator-src
+      resources:
+        limits:
+          memory: 6Gi
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: openshift-e2e-aws
```
Comment (markusthoemmes) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7-aws-mesh.yaml:212]: Should we add this to the "usual" 4.7 file? Seems unnecesary to duplicate everything.

**Comment by nak3** [comment]
/test pj-rehearse

One test `TestShouldHaveCgroupReadOnly` failed due to `x509: certificate signed by unknown authority`. It is the first time to see this error and not sure why only this test got the cert issue. Let's see another run.

```
    spoof.go:176: NOT Retrying http://should-have-cgroup-r-gulflobu-serving-tests.apps.ci-op-g2d8pxy9-cb7ce.origin-ci-int-aws.dev.rhcloud.com: Get "https://should-have-cgroup-r-gulflobu-serving-tests.apps.ci-op-g2d8pxy9-cb7ce.origin-ci-int-aws.dev.rhcloud.com/": x509: certificate signed by unknown authority
```

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7-aws-mesh.yaml (hunk 1 above)
Comment (nak3) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7-aws-mesh.yaml]: Yes! Updated it now.

File: ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7.yaml [hunk 2]
```diff
@@ -293,6 +293,50 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: openshift-e2e-aws
+- as: e2e-mesh-ocp-47-continuous
+  cron: 0 0 * * 0
```
Comment (markusthoemmes) [inline ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7.yaml:297]: Any reason this cron is different from the others? All others are `0 */12 * * 1-5`

Comment (nak3) [reply ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.7.yaml]: No, there is no reason. I just copied it from aws-ovn yaml and it has different cron schedule.
Updated.

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**